### DISCUSSION TITLE
https://theme-hope.vuejs.press/zh/get-started/

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#812] [Announcements]
# https://theme-hope.vuejs.press/zh/get-started/

一个具有强大功能的 vuepress 主题✨

https://theme-hope.vuejs.press/zh/get-started/

**Comment by Charlieconomy** [comment]
2024.4.10安装，从页面1使用链接跳转到页面2后，点击后退只能回到页面1的最顶端，无法回到页面1的原来的链接位置，请注意

**Reply by Charlieconomy** [reply]
并且无法使用链接转到目标页面书签处

**Comment by Tiang-88** [comment]
double（%lf）：数据范围$[-2^{1024}，2^{1024}]$，即$[-1.79*10^{308} , 1.79*10^{308}]$

**Comment by Chenzhexian** [comment]
打包部署后有俩个js报404，这个有办法解决吗

**Comment by SHIKEAIXY** [comment]
你好问一下404的页面要怎么改动提示呢
1. 页面不存在
2. 我们是怎么来到这儿的？
3. 看起来你访问了一个失效的链接

**Reply by Mister-Hope** [reply]
https://theme-hope.vuejs.press/zh/config/theme/i18n.html

`routeLocales.notFoundMsg`

**Comment by zxwin0125** [comment]
大佬，文档本地构建运行的时候如何优化下速度呢，我现在的博客运行起来要100s

**Comment by Utaoki-henji** [comment]
js文件全部报404,怎么处理

**Reply by Mister-Hope** [reply]
base 设置的不对
